1.0-draft/index.html

```diff
@@ -761,6 +763,8 @@ <h3>Suggest Responses</h3>
               <dd>Its corresponding human-readable name, to be displayed prominently to the user;</dd>
               <dt><code>description</code></dt>
               <dd>An optional description which can be provided to disambiguate namesakes, providing more context. This could for instance be displayed underneath the <code>name</code>;</dd>
+              <dt><code>image</code></dt>
+              <dd>An optional URL to an image illustrating the entity, helping the user identify the entity visually. This image could for instance be displayed alongside the <code>name</code> and <code>description</code>;</dd>
```

Comment:
Suggestion for consistency with the other changes in this PR:

```suggestion
              <dd>An optional URL of an image illustrating the entity, helping the user identify the entity visually. This image could for instance be displayed alongside the <code>name</code> and <code>description</code>;</dd>
```